lib/TraackrApi/TraackrApiObject.php

```diff
@@ -9,6 +9,25 @@
 
    private $curl;
 
+   // Headers passed with each request
+   private $curl_headers  = array(
```

Comment:
Sorry @KidA78, missed your comment, i was responding to @sprak3000.

I can't think of any header we would want to reject. Keep in mind that currently, someone can make direct curl calls and pass whatever headers they want. So it's something that should be handled by the API directly instead.